vicar/vicarlabel.py

```
@@ -522,14 +522,21 @@ def _interpret_source(self, source, required=False, fileio=False):
-            elif isinstance(source, pathlib.Path):
+            elif isinstance(source, (FCPath, Path)):
                 filepath = source
                 source = VicarLabel.read_label(source)
-            elif isinstance(source, str) and os.path.exists(source):
-                filepath = pathlib.Path(source)
-                source = VicarLabel.read_label(filepath)
+            elif isinstance(source, str):
+                filepath_temp = FCPath(source)
+                try:
+                    exists = filepath_temp.exists()
+                except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)